docs/getting-started.md

```diff
@@ -4,74 +4,119 @@ sidebar_position: 2
 
-   Add the following dependency to your `build.sbt` file:
+3. Add the dependency on OpenTelemetry autoconfiguration extension to your `build.sbt` file:
    ```scala
-   libraryDependencies += "io.scalac" %% "mesmer-akka-extension" % "0.7.0"
+   libraryDependencies ++= Seq(
+     "io.opentelemetry" % "opentelemetry-sdk-extension-autoconfigure" % "1.13.0-alpha"
+   )
    ```
 
-   Add this entry to your `application.conf`:
-   ```
-   akka.actor.typed.extensions = ["io.scalac.mesmer.extension.AkkaMonitoring"]
+4. Add an exporter for your favorite protocol to your `build.sbt` file:
+   
+   **For OTLP**
+   ```scala
+   libraryDependencies ++= Seq(
+     "io.opentelemetry" % "opentelemetry-exporter-otlp" % "1.13.0",
+   )
    ```
 
-2. Download [opentelemetry-javaagent.jar](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.12.0/opentelemetry-javaagent.jar) from `opentelemetry-java` Releases.
-
-3. Download [mesmer-otel-extension.jar](https://github.com/ScalaConsultants/mesmer/releases/download/v0.7.0/mesmer-otel-extension.jar) from `mesmer` Releases.
-
-4. Enable Prometheus exporter by adding the following dependencies to your `build.sbt` file:
+   **For Prometheus**
    ```scala
    libraryDependencies ++= Seq(
-     "io.opentelemetry" % "opentelemetry-sdk-extension-autoconfigure" % "1.13.0-alpha",
-     "io.opentelemetry" % "opentelemetry-exporters-prometheus"        % "0.9.1"
+     "io.opentelemetry" % "opentelemetry-exporter-prometheus" % "1.13.0-alpha".
    )
    ```
 
 5. Run the application with the following options:
    ```sh
    -javaagent:path/to/opentelemetry-javaagent.jar
    -Dotel.javaagent.extensions=mesmer-otel-extension.jar
+   # add the following option if you're using Prometheus exporter
+   -Dotel.metrics.exporter=prometheus
    ```
    Eg. if you're running the application with `sbt run` add this to `build.sbt` file:
    ```scala
    run / fork := true
    run / javaOptions ++= Seq(
      "-javaagent:path/to/opentelemetry-javaagent.jar",
-     "-Dotel.javaagent.extensions=path/to/mesmer-otel-extension.jar"
+     "-Dotel.javaagent.extensions=path/to/mesmer-otel-extension.jar",
+     // add the following option if you're using Prometheus exporter
+     "-Dotel.metrics.exporter=prometheus"
    )
    ```
    or if you're running your application as a `jar` from command line:
    ```sh
    java \
      -javaagent:path/to/opentelemetry-javaagent.jar \
      -Dotel.javaagent.extensions=path/to/mesmer-otel-extension.jar \
+     # add the following option if you're using Prometheus exporter
+     -Dotel.metrics.exporter=prometheus \
      -jar your-app.jar
    ```
 
-6. Test the Prometheus endpoint:
+6. Test it:
+   
+   **For OTLP**
+
+   You need to have a running OTLP Collector for the metrics to be automatically streamed to it.
+
+   **For Prometheus**
+
+   Call the metrics endpoint:
    ```sh
    curl -i http://localhost:9464
```

Comment:
Does this work? I usually just go to localhost:9090 and query for metrics there.

Reply:
but you have Prometheus running, right? localhost:9464 is the endpoint that Prometheus calls to get the metrics from the application